explorer/lib/explorer/contract_managers/avs_directory_manager.ex

```diff
@@ -14,7 +14,7 @@ defmodule AVSDirectoryManager do
 
     :error ->
       raise(
-        "Config file not read successfully, did you run make create-env? If you did,\n make sure Eigenlayer config file is correctly stored"
+        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure Eigenlayer config file is correctly stored"
```

Comment:
Here it is OK because avs_directory is an Eigenlayer contract

Reply:
Ok, ty for pointing that out!